### PR TITLE
Moves spring controllers to ulisboa specifications

### PR DESCRIPTION
**Pull request by joaofreitas-qubit**
Relates with: #qubIT-Omnis-5831

### CODE REVIEW

File: src/main/java/org/fenixedu/ulisboa/integration/sas/servlet/SasInitializer.java [hunk 1]
```diff
@@ -1,34 +0,0 @@
...
-import org.fenixedu.bennu.core.domain.Bennu;
-import org.fenixedu.ulisboa.integration.sas.domain.SchoolLevelTypeMapping;
-import org.fenixedu.ulisboa.integration.sas.domain.SocialServicesConfiguration;
-import org.fenixedu.ulisboa.integration.sas.service.sicabe.SicabeExternalService;
-
-import pt.ist.fenixframework.Atomic;
-import pt.ist.fenixframework.Atomic.TxMode;
-
-@WebListener
-public class SasInitializer implements ServletContextListener {
```
Comment (joaofreitas-qubit) [inline src/main/java/org/fenixedu/ulisboa/integration/sas/servlet/SasInitializer.java:16]: stays

File: src/main/java/org/fenixedu/ulisboa/integration/sas/tasks/sicabe/IngestSasScholarshipSicabe.java [hunk 2]
```diff
@@ -1,190 +0,0 @@
...
-import org.fenixedu.bennu.scheduler.annotation.Task;
-import org.fenixedu.ulisboa.integration.sas.domain.CandidacyState;
-import org.fenixedu.ulisboa.integration.sas.domain.SasScholarshipCandidacy;
-import org.fenixedu.ulisboa.integration.sas.domain.SocialServicesConfiguration;
-import org.fenixedu.ulisboa.integration.sas.service.sicabe.SicabeExternalService;
-
-import pt.ist.fenixframework.Atomic;
-import pt.ist.fenixframework.FenixFramework;
-
-//Force task to be read only and process each report on its own transaction to avoid errors in a report affecting other reports
```
Comment (joaofreitas-qubit) [inline src/main/java/org/fenixedu/ulisboa/integration/sas/tasks/sicabe/IngestSasScholarshipSicabe.java:32]: stays

File: src/main/java/org/fenixedu/ulisboa/integration/sas/service/sicabe/SicabeExternalService.java [hunk 3]
```diff
@@ -1,984 +0,0 @@
...
-import pt.dges.schemas.data.sicabe.v1.RegistarMatriculaAlunoRequest;
-import pt.dges.schemas.data.sicabe.v1.ResultadoEstadoCandidatura;
-import pt.dges.schemas.data.sicabe.v1.TipoDocumentoIdentificacao;
-import pt.dges.schemas.data.sicabe.v1.TipoRegime;
-import pt.dges.schemas.services.sicabe.v1.DadosAcademicos;
-import pt.dges.schemas.services.sicabe.v1.DadosAcademicosAlterarDadosAcademicosPrimeiraVezSicabeBusinessMessageFaultFaultMessage;
-import pt.dges.schemas.services.sicabe.v1.DadosAcademicosAlterarDadosAcademicosPrimeiraVezSicabeErrorMessageFaultFaultMessage;
-import pt.dges.schemas.services.sicabe.v1.DadosAcademicosAlterarDadosAcademicosPrimeiraVezSicabeValidationMessageFaultFaultMessage;
-import pt.dges.schemas.services.sicabe.v1.DadosAcademicosAlterarDadosAcademicosRestantesCasosSicabeBusinessMessageFaultFaultMessage;
-import pt.dges.schemas.services.sicabe.v1.DadosAcademicosAlterarDadosAcademicosRestantesCasosSicabeErrorMessageFaultFaultMessage;
```
Comment (joaofreitas-qubit) [inline src/main/java/org/fenixedu/ulisboa/integration/sas/service/sicabe/SicabeExternalService.java:66]: stays

File: src/main/java/org/fenixedu/ulisboa/integration/sas/util/SasPTUtil.java [hunk 4]
```diff
@@ -1,50 +0,0 @@
...
- * You should have received a copy of the GNU Lesser General Public License
- * along with FenixEdu Treasury.  If not, see <http://www.gnu.org/licenses/>.
- */
-package org.fenixedu.ulisboa.integration.sas.util;
-
-import org.fenixedu.bennu.SasSpringConfiguration;
-import org.fenixedu.bennu.core.i18n.BundleUtil;
-import org.fenixedu.commons.i18n.LocalizedString;
-
-public class SasPTUtil {
```
Comment (joaofreitas-qubit) [inline src/main/java/org/fenixedu/ulisboa/integration/sas/util/SasPTUtil.java:32]: stays

File: pom.xml [hunk 5]
```diff
@@ -151,39 +151,37 @@
 					</execution>
```
Comment (pabrantes) [inline pom.xml:151]: @joaofreitas-qubit let's just checkout this file directly from master so it stops being seen as a change when it's only spacing.

**Review by pabrantes — CHANGES_REQUESTED** [review]

**Review by pabrantes — APPROVED** [review]

**Pull request merged**